src/nvim/ex_getln.c

```diff
@@ -2342,8 +2361,7 @@ enum { MAX_CB_ERRORS = 1 };
 /// Should use built-in command parser or user-specified one. Currently only the
 /// latter is supported.
 ///
-/// @param[in]  colored_ccline  Command-line to color.
-/// @param[out]  ret_ccline_colors  What should be colored. Also holds a cache:
+/// @param[in/out]  colored_ccline  Command-line to color. Also holds a cache:
 ///                                 if ->prompt_id and ->cmdbuff values happen
 ///                                 to be equal to those from colored_cmdline it
 ///                                 will just do nothing, assuming that ->colors
```

Comment:
Is the comment still correct? `colored_ccline` holds a cache?

Reply:
Also `in/out` is written `in,out`. Don’t know whether doxygen is recognizing one, other or both, but it is how it is written in the rest of the code base.

Reply:
@justinmk yes, if the cmdline is not changed, it reuses the old color .
@ZyX-I noted, I might have copied some non-doxygen docstring by accident.